src/frontend/screens/Audio/AudioPlaybackScreen.tsx

```diff
@@ -0,0 +1,214 @@
+} from '../../lib/styles.ts';
+import {StatusBar} from 'expo-status-bar';
+import React from 'react';
+import {Duration} from 'luxon';
+import PlayArrow from '../../images/playArrow.svg';
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons';
+import {AudioStackParamList} from '../../sharedTypes/navigation.ts';
+import {useAudioPlayback} from '../../hooks/useAudioPlayback.ts';
+import {useFocusEffect} from '@react-navigation/native';
+import {useNavigationFromRoot} from '../../hooks/useNavigationWithTypes.ts';
+import MaterialIcon from 'react-native-vector-icons/MaterialIcons';
+import {AnimatedTimer} from './AnimatedTimer.tsx';
+import {AnimatedProgressBar} from './AnimatedProgressBar.tsx';
+import {useDerivedValue, withTiming} from 'react-native-reanimated';
+import {NativeStackScreenProps} from '@react-navigation/native-stack';
+import {useBottomSheetModal} from '../../sharedComponents/BottomSheetModal/index.tsx';
+import {AudioRecordingDeleteBottomSheet} from './bottomSheet/AudioRecordingDeleteBottomSheet.tsx';
+import {AudioRecordingSuccessBottomSheet} from './bottomSheet/AudioRecordingSuccessBottomSheet.tsx';
+import {useDraftObservation} from '../../hooks/useDraftObservation.ts';
+import {AudioShareBottomSheet} from './bottomSheet/AudioShareBottomSheet.tsx';
+
+export const AudioPlaybackScreen: React.FC<
+  NativeStackScreenProps<AudioStackParamList, 'Playback'>
+> = params => {
+  const {
+    sheetRef: deleteAudioSheetRef,
+    isOpen: isOpenDeleteBottomSheet,
+    closeSheet: closeDeleteBottomSheet,
+    openSheet: openDeleteBottomSheet,
+  } = useBottomSheetModal({
+    openOnMount: false,
+  });
+
+  const {
+    sheetRef: successSheetRef,
+    isOpen: isOpenSuccessSheet,
+    openSheet: openSuccessSheet,
+  } = useBottomSheetModal({
+    openOnMount: false,
+  });
+  const {
+    sheetRef: shareSheetRef,
+    isOpen: isOpenShareSheet,
+    openSheet: openShareSheet,
+    closeSheet: closeShareSheet,
+  } = useBottomSheetModal({
+    openOnMount: false,
+  });
+
+  const navigation = useNavigationFromRoot();
+  const {recordingUri, previewOnly} = params.route.params;
+  const observation = useDraftObservation();
+
+  const {
+    isReady,
+    isPlaying,
+    currentPosition,
+    duration,
+    startPlayback,
+    stopPlayback,
+  } = useAudioPlayback(recordingUri);
+
+  useFocusEffect(() => {
+    navigation.setOptions({
+      // eslint-disable-next-line react/no-unstable-nested-components -- it's supported way to do it like that
+      headerLeft: () => (
+        <MaterialIcon
+          name="close"
+          size={25}
+          color={WHITE}
+          onPress={() => {
+            if (previewOnly) {
+              navigation.goBack();
+            } else {
+              observation.addAudioRecording({
+                duration,
+                uri: recordingUri,
+                createdAt: new Date().getTime(),
+              });
+              openSuccessSheet();
+            }
+          }}
+        />
```

Comment:
Maybe it's better if we create a component for that left button and also create a function to handle that:
```
onPress={() => {
  if (previewOnly) {
    navigation.goBack();
  } else {
    observation.addAudioRecording({
      duration,
      uri: recordingUri,
      createdAt: new Date().getTime(),
    });
    openSuccessSheet();
  }
}}
```